src/service/PropertyService.ts

```diff
@@ -45,7 +49,8 @@ export const getAllWishListService = async () => {
   }
 }
 
+
 export async function getPropertyById(propertyId:number) {
-      const res = await axios.get(`/properties/${propertyId}`);
-      return res.data.metaData
-}
+  const res = await axios.get(`/properties/${propertyId}`);
```

Comment:
use axiosInstance

Reply:
we just use it in case we wanna attach the bearer authorization token. So in this case, we don't need the bearer so we use a global axios.